go-controller/pkg/libovsdbops/switch.go

```diff
@@ -224,11 +227,17 @@ func createOrUpdateLogicalSwitchPortsOps(nbClient libovsdbclient.Client, ops []l
 	for i := range lsps {
 		lsp := lsps[i]
 		opModel := operationModel{
-			Model:          lsp,
-			OnModelUpdates: onModelUpdatesAll(),
-			DoAfter:        func() { sw.Ports = append(sw.Ports, lsp.UUID) },
-			ErrNotFound:    false,
-			BulkOp:         false,
+			Model: lsp,
+			OnModelUpdates: []interface{}{
+				&lsp.Addresses,
+				&lsp.Type,
+				&lsp.TagRequest,
+				&lsp.Options,
+				&lsp.PortSecurity,
```

Comment:
done

Reply:
actually I dont think it makes sense here as it is iterating over multiple switch ports

Reply:
Ok, I think we should then take care to have all fields we could ever be interesting in updating in this list.

I think it would fit to have a getAllUpdateableLogicalSwitchPortFields method in model.go (where we are putting stuff we hope to be able to generate one day) with this field list.

Also maybe update the function doc text to explicitly state all fields are updated?